### PR TITLE
Add Alpaka Process Modifier

### PR DESCRIPTION
**Pull request by jsamudio**
#### PR description:

This PR is made to introduce a process modifier, `alpaka`, which will be utilized to enable Alpaka based modules such as those in #43130 and #41117. This PR will simply add the process modifier, but no changes to any workflows are introduced.

#### PR validation:

Basic battery of tests passed:
`runTheMatrix.py -l limited -i all --ibeos`

In the future this process modifier will be used to enable the validation of Alpaka modules.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

PR is not a backport.

@fwyzard

### CODE REVIEW

File: Configuration/StandardSequences/python/Services_cff.py [hunk 1]
```diff
@@ -13,9 +13,15 @@
 def _addProcessAccelerators(process):
     process.load("Configuration.StandardSequences.Accelerators_cff")
 
+def _addProcessAcceleratorsAlpaka(process):
+    process.load("HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cfi")
```
Comment (makortel) [inline Configuration/StandardSequences/python/Services_cff.py:17]: I would prefer the loading of `ProcessAcceleratorAlpaka_cfi` to be in `Accelerators_cff`. I also think that, given the state of the GPU matters in the offline side, the import there could be unconditional.

Comment (makortel) [reply Configuration/StandardSequences/python/Services_cff.py:17]: Yes.

I think adding the `alpaka` modifier as one of the ways to call `makeProcessModifier(_addProcessAccelerators)` below is fine (or, I'm not against).

**Comment by makortel** [comment]
enable gpu

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by antoniovilela** [comment]
@makortel 
Are you happy with this PR?
Thanks.

**Comment by makortel** [comment]
Yes, it looks fine as far as I'm concerned.

**Comment by antoniovilela** [comment]
> Yes, it looks fine as far as I'm concerned.

Thanks.

**Comment by antoniovilela** [comment]
+1

**Pull request merged**